app/models/like.rb

```diff
@@ -1,4 +1,16 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :increment_post_like_counter
```

Comment:
**OPTIONAL** - I suggest calling your `increment` method after the data has been saved to the database, this would help ensure that our data was saved before any increment. Please fix this 🙏

Reply:
done